### DISCUSSION TITLE
Drafts (note-taking app)

### DISCUSSION THREAD

**Discussion by cmdkev** [catppuccin/catppuccin#2047] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

[Drafts docs](https://docs.getdrafts.com/docs/extending/development/theme-format)

[Drafts tools for making themes](https://tools.getdrafts.com/themes/theme-builder)

[Blog post on how to make themes](https://nahumck.me/drafts-26-the-customization-update/)

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by romanskorobagatko** [comment]
FlohGro made [it](https://actions.getdrafts.com/t/2Vp).
